libexec/scoop-update.ps1

```diff
@@ -331,9 +331,14 @@ if (!$apps) {
         }
     }
 
-    $suggested = @{};
+    $suggested = @{ };
     # $outdated is a list of ($app, $global) tuples
-    $outdated | ForEach-Object { update @_ $quiet $independent $suggested $use_cache $check_hash }
+    $outdated | ForEach-Object { try {
+            update @_ $quiet $independent $suggested $use_cache $check_hash
+        } catch {
+            Write-Error $_
```

Comment:
```suggestion
            error $_
```

Reply:
What do you want to show in this error? Name of manifest or error message?

Reply:
I want show error including error message for report issue.